book/src/zcash/ywallet-demo.md

```diff
@@ -20,10 +28,26 @@ Install `cargo` and `git`.
 Clone the repositories:
 
 ```
-git clone --branch add-redpallas https://github.com/ZcashFoundation/frost-zcash-demo.git
-git clone --branch frost-demo https://github.com/ZcashFoundation/zwallet.git
+git clone https://github.com/ZcashFoundation/frost-zcash-demo.git
+git clone --recurse-submodules --branch frost-demo https://github.com/ZcashFoundation/zwallet.git
+git clone https://github.com/ZcashFoundation/zcash.git
+```
+
+Then:
+
+```
+./zcash/zcutil/fetch-params.sh
 ```
 
+
+Download sprout and sapling:
+
+[Sprout params](https://download.z.cash/downloads/sprout-groth16.params)
+
+[Sapling-spend params](https://download.z.cash/downloads/sapling-spend.params)
+
+[Sapling-output params](https://download.z.cash/downloads/sapling-output.params)
+
```

Comment:
These are redundant - either the user run `fetch-params.sh`, or they download the parameters. Since the script only works on Linux it seems better to just instruct them to download the parameters.

This requires describing where to download the parameters, i.e. `~/.zcash-params/` on Linux.

Reply:
Resolved in [5f5f46a](https://github.com/ZcashFoundation/frost/pull/562/commits/5f5f46a2a32583c574235460ff19e775e7eff396)

Reply:
I think we still need to explain where to download the files